tsconfig.json

```diff
@@ -1,6 +1,5 @@
 {
   "compilerOptions": {
-    "allowJs": true,
```

Comment:
I know we removed this when pair programming @TeresaGobble, but let's just double check and make sure there is no adverse effects by removing this.

Reply:
Yeah you should restore this back.